pvr.vuplus/resources/language/resource.language.de_de/strings.po

```diff
@@ -116,6 +116,10 @@ msgctxt "#30028"
 msgid "Use Secure HTTP (https)"
 msgstr "Sicheres HTTP (https) verwenden"
 
+msgctxt "#30030"
```

Comment:
@Jalle19 Is it ok to amend the commit to undo this (and thereby change the history)?